api/bertybridge.proto

```diff
@@ -4,9 +4,20 @@ package berty.bridge.v1;
 
 import "errcode.proto";
 
-option go_package = "berty.tech/berty/go/framework/bertybridge/internal/bridgepb";
+option go_package = "berty.tech/berty/go/pkg/bertybridge";
```

Comment:
```suggestion
option go_package = "berty.tech/berty/go/pkg/messengerbridge";
```

or maybe better, since this lib is not only used as a bridge:

```suggestion
option go_package = "berty.tech/berty/go/pkg/accountmanager";
```